mobile/src/screens/meditation/components/controls/components/button/button/button.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { type SvgProps } from 'react-native-svg';
+
+import { TouchableOpacity } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
+
+import { styles } from './styles';
+
+type Properties = {
+  onPress: () => void;
+  Icon: React.FC<SvgProps>;
+  isRounded?: boolean;
+};
+
+const Button: React.FC<Properties> = ({ onPress, Icon, isRounded = false }) => {
+  return (
+    <TouchableOpacity onPress={onPress} style={isRounded && styles.rounded}>
+      <Icon color={isRounded ? AppColor.BLUE_200 : AppColor.GRAY_400} />
```

Comment:
It can be misunderstanding that `isRounded` flag change the color of the icon. Also we already have common component with `Button` name. If it's really necessary to make separate component, maybe it's better to name it like `MeditationControlButton` or smth like that?

Reply:
isRounded changes the color because this way I indicate that this button is central (in the design, it's blue).
I don't see the point in complicating component names